usagov/vote-gov-drupal
# Testing Role Access to Content

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
[vote-919](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-919&assignee=611272e0eb7aef00694b8a97)
[vote-918](https://bixal-projects.atlassian.net/jira/software/c/projects/VOTE/boards/254?modal=detail&selectedIssue=VOTE-918&assignee=611272e0eb7aef00694b8a97)
## Description (optional)
This PR will add test coverage for both Site admin and content Manager rolls access.  This will go through creating pages, checking if they can view drafts can move things out of draft etc. 

## Deployment and testing (required, if applicable)
### Testing steps
1. Pull down branch 
2. start local 
3. cd into `testing`
4. run `npm run cy:createUser`
5. run `npm run cy:open` and select the `user-content-access.cy.js` file and verify that test is passing 
6. once the test is complete be sure to run `npm run cy:deleteUser` back in the terminal in the `testing` folder
